code/datums/components/crafting/recipes.dm

```diff
@@ -404,6 +404,14 @@
 	time = 25
 	category = CAT_MISC
 
+/datum/crafting_recipe/helm
```

Comment:
This has the same datum name as the recipe below it, so it gets overwritten by that one, making this recipe uncraftable.